addons/web_tour/static/src/debug/tour_dialog_component.xml

```diff
@@ -11,6 +11,9 @@
             <t t-set="caption">Testing tours</t>
             <t t-set="tours" t-value="testingTours"/>
         </t>
+        <t t-set-slot="footer">
+            <button class="btn btn-primary o-default-button" t-on-click="() => props.close()">Close</button>
```

Comment:
It feels like we will want to support a "closeLabel" prop at some point, but no need to do it right away, keep what you did.